frontend/src/Series/Edit/EditSeriesModalContentConnector.js

```diff
@@ -80,7 +84,20 @@ class EditSeriesModalContentConnector extends Component {
   // Listeners
 
   onInputChange = ({ name, value }) => {
-    this.props.dispatchSetSeriesValue({ name, value });
+    const {
+      dispatchSetSeriesValue,
+      isWindows,
+      title,
+      item: { path }
+    } = this.props;
+
+    dispatchSetSeriesValue({ name, value });
+
+    // Also update the path if the root folder path changes
+    if (name === 'rootFolderPath' && value !== 'noChange' && path.value.indexOf(value) === -1) {
+      const newPath = combinePath(isWindows, value, [title]);
```

Comment:
Using the series title isn't sufficient, this path needs to be generated on the backend to take the format into account.